src/containers/search-result-page/assets/js/script.js

```diff
@@ -1,22 +1,68 @@
 const APIKey = "612305371d20a404c362f2768ab01667";
 
-yesGenresLocalStorage = [""];
-noGenresLocalStorage = [""];
-moviesArray = [""];
-genresArray = [""];
 
-var queryURL = "https://api.themoviedb.org/3/discover/movie?api_key=b574c405150412e74904c09bd2c259b6&language=en-US&sort_by=popularity.desc&include_adult=false&include_video=false&page=1&with_watch_monetization_types=flatrate";
+var yesGenresLocalStorage = [""];
+var noGenresLocalStorage = [""];
+var keywords = "";
+var language = "en-US";
+var moviesArray = [""];
+var genresArray = [""];
+var today = moment();
+var toYear = moment(today).format("YYYY");
+var fromYear = toYear;
+
+
+var queryURL = "";
 var genresQueryURL = "https://api.themoviedb.org/3/genre/movie/list?api_key=" + APIKey + "&language=en-US";
 
 function readLocalStorage() {
     yesGenresLocalStorage = JSON.parse(localStorage.getItem("yesGenres"));
     noGenresLocalStorage = JSON.parse(localStorage.getItem("noGenres"));
+    keywords = localStorage.getItem("keywords");
+    language = localStorage.getItem("language");
+}
+
+function arrayToString(myArray) {
+    var myString = "";
+    for (var index = 0; index < myArray.length; index++) {
+        var element = myArray[index];
+        if (!myString) {
+            myString = element;
+        } else
+            myString = myString + "%2C" + element;
+    }
+    return (myString);
+}
 
+function buildQueryURL() {
+    queryURL = "https://api.themoviedb.org/3/discover/movie?api_key=" + APIKey;
+
+    language = "&language=" + language;
```

Comment:
we should be using `&with_original_language` instead `&language`

Reply:
got it
Let me fix

Reply:
Done